reactions/src/HALO.cpp

```diff
@@ -784,16 +797,17 @@ else{
 // cbv terms
   psptcbv = sqrt(pspt*plnu);
 
-// reactionm @ k0 with 2halo term = 1 loop spt
+// reaction  @ k0 with 2halo term = 1 loop spt
   double rsptk0 = (fvt2 * pspt + 2.*fv*fvt*psptcbv + fv2*plnu)/(psptp);
 
 
   double prefac = 1./((bigE-1.)*plcb*fvt2);
   double term1 = p1h*fvt2 + (fvt2*bigE*plcb - fv2*plnu - (p1hp+plm)*rsptk0);
-  double term2 = 2.*sqrt(fvt2*fv2*(p1hp+plm)*plnu*rsptk0);
+//  double term2 = 2.*sqrt(fvt2*fv2*(p1hp+plm)*plnu*rsptk0); // suspected typo -- fvt2 inclusion
+  double term2 = 2.*sqrt(fv2*(p1hp+plm)*plnu*rsptk0);
 
    argument = prefac * (term1 + term2); // 1st root
-// argument = prefac * (term1 - term2); // 2nd root
+  //   argument = prefac * (term1 - term2); // 2nd root
```

Comment:
1st root is correct

Reply:
should you add this comment on the actual code?